Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/AmbiguityResolutionAlgorithm.hpp

```diff
@@ -37,6 +37,9 @@ class AmbiguityResolutionAlgorithm final : public BareAlgorithm {
 
     /// Maximum amount of shared hits per track.
     std::uint32_t maximumSharedHits = 1;
+
+    /// Minumum number of measurement to form a track.
+    size_t nMeasurementsMin = 7;
```

Comment:
this is the change which causes physmon to create a diff, right @Corentin-Allaire ? cc @paulgessinger

Reply:
Yes ! As I mention before In my opinion with the geometry of the ODD a track with < 7 measurement is very unlikely to be a good track